homeassistant/components/sum/sensor.py

```diff
@@ -0,0 +1,226 @@
+"""Support for displaying sum values."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import (
+    PLATFORM_SCHEMA as PARENT_PLATFORM_SCHEMA,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_UNIT_OF_MEASUREMENT,
+    CONF_NAME,
+    CONF_UNIQUE_ID,
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+)
+from homeassistant.core import Event, HomeAssistant, State, callback
+from homeassistant.helpers import config_validation as cv, entity_registry as er
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.event import async_track_state_change_event
+from homeassistant.helpers.reload import async_setup_reload_service
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType, EventType
+
+from .const import CONF_ENTITY_IDS, CONF_ROUND_DIGITS, DEFAULT_NAME, DOMAIN, PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = "mdi:calculator"
+ATTR_ENTITIES = "entities"
+ATTR_VALID_ENTITIES = "valid_entities"
+ATTR_COUNT_VALID = "count_valid"
+
+PLATFORM_SCHEMA = PARENT_PLATFORM_SCHEMA.extend(
```

Comment:
This should not be using a platform schema (ADR-0007)

Reply:
Right. I though maybe helpers was seen differently but wasn't checking the ADR.
Will change this